src/vma/util/utils.cpp

```diff
@@ -1035,6 +1035,37 @@ int validate_ipoib_prop(const char* ifname, unsigned int ifflags,
 	}
 }
 
+//NOTE RAW_QP_PRIVLIGES_PARAM_FILE does not exist on ofed upstream
+int validate_raw_qp_privliges()
+{
+	// RAW_QP_PRIVLIGES_PARAM_FILE: "/sys/module/ib_uverbs/parameters/disable_raw_qp_enforcement"
+	char raw_qp_privliges_value = 0;
+	if (priv_read_file((const char*)RAW_QP_PRIVLIGES_PARAM_FILE, &raw_qp_privliges_value, 1, VLOG_DEBUG) <= 0) {
+		return -1;
+	}
+	if (raw_qp_privliges_value != '1') {
+		return 0;
+	}
+	return 1;
+}
+
+bool check_user_capabilities() 
```

Comment:
maybe function name should be called: `verify_user_has_CAP_NET_RAW()`
we're not checking for any other cap bits.